src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+This package has [not] been tested [much] with Python 2.7, 3.4 and 3.5.
```

Comment:
@djyou you are right. It was more a general reminder to the team that we really should remove that everywhere (especially the example package!)